subprojects/core-api/src/main/java/org/gradle/BuildAdapter.java

```diff
@@ -27,6 +27,10 @@
     public void buildStarted(Gradle gradle) {
     }
 
+    @Override
+    public void beforeSettings(Settings settings) {
+    }
+
```

Comment:
Is this required given that the interface has a default implementation for this method?

Reply:
It's not, but I thought it best to add it for consistency.